lang/java/tools/pom.xml

```diff
@@ -145,7 +145,7 @@
     </dependency>
     <dependency>
       <groupId>org.apache.hadoop</groupId>
-      <artifactId>hadoop-core</artifactId>
```

Comment:
I guess hadoop-core is not required for hadoop2 profile, it is required for hadoop1, and we are building avro-1.7.7 for hadoop2.

I was following below commit url to make the changes 
https://github.com/ibmsoe/avro/commit/2b897f0020fbec49b1ac150a0bd1bfe00e6b07ef

But to be on safer side I will add both hadoop-core and hadoop-client as the dependencies.